citesphere/src/main/java/edu/asu/diging/citesphere/web/user/AddItemController.java

```diff
@@ -26,24 +28,32 @@
 
 @Controller
 @PropertySource("classpath:/config.properties")
+@PropertySource("classpath:/creators.properties")
```

Comment:
I believe there is an annotation PropertySources that you can give a list of files

Reply:
actually, do you use that?